client/src/lib/dependency-injection/graphql/buildReadOneQuery.js

```diff
@@ -1,9 +1,9 @@
 import { defaultTag } from './tags';
-import { getSingularName, getFields, getFragments } from './helpers';
+import { getSingularName, getIdOnlyVariables, getFields, getFragments, getIdOnlyParams } from './helpers';
 
 const buildReadOneQuery = (tag = defaultTag) => (
-  tag`query ReadOne${getSingularName}($ID: ID!)  {
-    readOne${getSingularName}(ID: $ID) {
+  tag`query ReadOne${getSingularName}${getIdOnlyVariables}  {
+    readOne${getSingularName}${getIdOnlyParams} {
```

Comment:
Turning `$ID: ID!` and `ID: $ID` into default fallbacks, which can be overridden by setting explicit params. See the new related test for an example.